### PR TITLE
[Datafactory]SDK Change for SapOpenHub and Rest

### PR DESCRIPTION
**Pull request by wenbof-zz**

### CODE REVIEW

File: src/SDKs/DataFactory/Management.DataFactory/changelog.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
 # Changelog for the Azure Data Factory V2 .NET SDK
 
 ## Current version
+## Version 3.0.2
+
+###  Feature Additions
+    - Add support for RestService Source
```
Comment (hvermis) [inline src/SDKs/DataFactory/Management.DataFactory/changelog.md]: Please fix the indentations

**Review by hvermis — APPROVED** [review]

File: src/SDKs/DataFactory/Management.DataFactory/changelog.md (hunk 1 above)
Comment (wenbof) [reply src/SDKs/DataFactory/Management.DataFactory/changelog.md]: thanks.

**Review by dsgouda — CHANGES_REQUESTED** [review]
How was this code generated? (using generate.ps1?) Please commit all changes generated including those to .txt files

**Comment by dsgouda** [comment]
@wenbof please link the Rest spec PR too

**Comment by hvermis** [comment]
@dsgouda This is the spec PR for this change:
https://github.com/Azure/azure-rest-api-specs/pull/5195

**Comment by shahabhijeet** [comment]
@wenbof can you answer the question raised by @dsgouda? you are missing meta data in this PR.

**Comment by dsgouda** [comment]
@wenbof please regenerate the code using generate.ps1 and commit changes to the .txt and SdkInfo*_.cs files

**Review by dsgouda — APPROVED** [review]
LGTM based on offline conversation with @hvermis

**Pull request merged**